docs/reference/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -227,7 +230,85 @@ GET my_index/_ilm/explain
 // CONSOLE
 // TESTRESPONSE[skip:no way to know if we will get this response immediately]
 
-Updating `my_executing_policy` to have no rollover action and, instead, go directly into a newly introduced `warm` phase.
+We can update `my_executing_policy` to enter the hot phase after one day.
+
+[source,js]
+------------------------
+PUT _ilm/policy/my_executing_policy
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "min_age": "1d",
+        "actions": {
+          "rollover": {
+            "max_docs": 1
+          }
+        }
+      },
+      "delete": {
+        "min_age": "10d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+// TEST[continued]
+
+Now, version 2 of this policy has `min_age` for the hot phase set to one day, but if we
+run the Explain API again, we will see that nothing has changed. The index `my_index`
+is still executing version 1 of the policy.
```

Comment:
Actually, one other comment: Maybe phrase this like "The index `my_index` has already entered the `hot` phase, so it will still use version 1 of the policy until it completes the `hot` phase."

Just seems a little clearer in terms of the point we're trying to make here.